charts/stable/nginx-ingress/values.yaml.gotmpl

```diff
@@ -1,7 +1,7 @@
 controller:
   replicaCount: 3
-  extraArgs:
```

Comment:
we still need the `extraArgs` when TLS is enabled to set the `default-ssl-certificate` flag

Reply:
oops :slightly_smiling_face:

Reply:
I've added it back, but under `tls.enabled` condition